src/_extensions/branding-guidelines.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Branding Guidelines
+authors:
+- shwetank-dixit
+intro: 'This lists the branding guidelines for using Opera trademarks for your applications.'
+---
+
+## Using the Opera add-ons badge
+
+Share the word that your app, extension or theme is available in the [Opera add-ons catalog](https://addons.opera.com) by using the "Get it from Opera add-ons" badge on your site.
+
+It could also be a good idea to use [inline installation](/extensions/inline-installation) along with this badge on your website. 
+
+<figure block="figure">
+<img elem="media" src="{{ page.url }}addons_206x58_en@2x.png" alt="Opera add-ons badge" width="206">
+</figure>
+
+While pre-approval is not required for you to use this badge, be sure that your use of the badge is consistent with the Opera add-ons [terms of use](https://addons.opera.com/developer/terms/) and these branding guidelines; otherwise, we may request that you modify or cease your use of the badge.
+
+### Badges (Multiple Languages)
+
+[Download the badge in multiple languages](opera_addons_badge_multiple_languages.zip) (English, Russian, German, Ukranian, French and Polish). Badges for each of these languages are available in multiple sizes.
+
+If you wan to download the Opera add-ons badge just for a particular language only, then please look below.
+
+### English
+
+<figure block="figure">
+<img elem="media" src="{{ page.url }}addons_206x58_en@2x.png" alt="Opera add-ons badge in English" width="206">
+</figure>
+
+[Download](opera_addons_badge_en.zip) the set of badges for English in various sizes.
+
+### Russian
+
+<figure block="figure">
+<img elem="media" src="{{ page.url }}addons_206x58_ru@2x.png" alt="Opera add-ons badge in Russian" width="206">
+</figure>
+
+[Download](opera_addons_badge_ru.zip) the set of badges for Russian in various sizes.
+
+### German
+
+<figure block="figure">
+<img elem="media" src="{{ page.url }}addons_206x58_de@2x.png" alt="Opera add-ons badge in German" width="206">
+</figure>
+
+[Download](opera_addons_badge_de.zip) the set of badges for German in various sizes.
+
+### Ukranian
+
+<figure block="figure">
+<img elem="media" src="{{ page.url }}addons_206x58_uk@2x.png" alt="Opera add-ons badge in Ukranian" width="206">
+</figure>
+
+[Download](opera_addons_badge_uk.zip) the set of badges for Ukranian in various sizes.  
+
+### French
+
+<figure block="figure">
+<img elem="media" src="{{ page.url }}addons_206x58_fr@2x.png" alt="Opera add-ons badge in French" width="206">
+</figure>
+
+[Download](opera_addons_badge_fr.zip) the set of badges for French in various sizes.
+
+### Polish
+
+<figure block="figure">
+<img elem="media" src="{{ page.url }}addons_206x58_pl@2x.png" alt="Opera add-ons badge in Polish" width="206">
+</figure>
+
+[Download](opera_addons_badge_pl.zip) the set of badges for Polish in various sizes.
```

Comment:
Could you please start all links to article’s resources from `{{ pade.id }}`? Otherwise they will be broken in RSS.